doc/source/cluster/kubernetes/k8s-ecosystem/volcano.md

```diff
@@ -236,10 +239,10 @@ kubectl get pods
 
 # NAME                                            READY   STATUS         RESTARTS   AGE
 # test-cluster-0-worker-worker-ddfbz              1/1     Running        0          7m
-# test-cluster-0-head-vst5j                       1/1     Running        0          7m
+# test-cluster-0-head                             1/1     Running        0          7m
 # test-cluster-0-worker-worker-57pc7              1/1     Running        0          6m59s
 # test-cluster-1-worker-worker-6tzf7              0/1     Pending        0          2m12s
-# test-cluster-1-head-6668q                       0/1     Pending        0          2m12s
+# test-cluster-1-head                             0/1     Pending        0          2m12s
 # test-cluster-1-worker-worker-n5g8k              0/1     Pending        0          2m12s
 ```
```

Comment:
There is no trailing hash on the head pods.
<img width="970" alt="volcano-trailing-hash-01" src="https://github.com/user-attachments/assets/46dc45ea-0edf-4903-a462-4b33b0bfa55d" />